topics/admin/tutorials/tool-management/slides.html

```diff
@@ -723,6 +723,25 @@
 
 ---
 
+## Suite repositories
+
+- Suites can be used to split up a set of tools into multiple repositories
+  - A suite is a single repository that 'depends' on many others
+  - When you install the suite, all 'dependency repositories' will be installed too
+- With a tweak to `.shed.yml`, [Planemo](https://planemo.readthedocs.io) can upload a directory of tools to the TS as a suite of separate tools
+- Examples: suite_samtools, suite_hicexplorer
+
+???
+
+There are also repositories in the Tool Shed that are suite repositories.  These can be used
```

Comment:
Would you mind making each sentence a bullet point? Otherwise this looks ok so far.

Reply:
Thanks @hexylena. I've updated the speaker notes (there were quite a few typos) and split up the sentences to make it more readable.

Reply:
awesome!